lib/commands/update.js

```diff
@@ -114,7 +114,9 @@ class UpdateCommand extends Command {
                     return;
                 }
 
-                return this.ui.listr(tasks, context);
```

Comment:
Note to mysql: `ghost update --force` should update to 2.0 if you are on the latest version. This is currently not the case, it always installs the latest v1 release. Will open a PR soon.

Reply:
Fixed https://github.com/TryGhost/Ghost-CLI/pull/773/commits/435ad4d1e761e0715d15c0e8c50632691a08f961